software/specifically-clementines.yml

```
@@ -0,0 +1,15 @@
+name: "Specifically Clementines"
+website_url: "https://davideshay.github.io/groceries/"
+source_code_url: "https://github.com/davideshay/groceries"
+description: "Grocery shopping app previously called 'groceries'. Reliable sync with multiple users/devices (web/mobile/Android/iOS). Available on Google Play Store. Includes recipes and integration with Tandoor."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Javascript
+  - Nodejs
```

```suggestion
platforms:
  - Docker
```

Docker is the only supported installation method (https://davideshay.github.io/groceries/installation/installation/)

> Values for platform are the main server-side requirements for the software